app/Broadcasting/GeneralChannel.php

```
@@ -0,0 +1,37 @@
+        ];
+    }
+
+    public function broadcastWith(): array
+    {
+        return ['message' => $this->message];
+    }
+
+    /**
+     * Authenticate the user's access to the channel.
+     */
+    public function join(User $user): array|bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)